pkg/kubelet/kubelet.go

```diff
@@ -2346,10 +2347,13 @@ func (kl *Kubelet) setNodeStatus(node *api.Node) error {
 	// Check whether network is configured properly
 	networkConfigured := kl.doneNetworkConfigure()
 
+	selinuxRequired := capabilities.Get().EnableSELinuxIntegration
+	selinuxOK := (selinuxRequired && kl.selinuxEnabled()) || !selinuxRequired
```

Comment:
If SELinux integration isn't enabled and SELinux is enabled on the host, I think that should be an error.